gratipay/models/participant/email.py

```diff
@@ -206,8 +215,8 @@ def remove_email(self, address):
 
 
     def queue_email(self, spt_name, **context):
```

Comment:
Fair enough. Turns out this will take some further refactoring, though. Forthcoming ...